.github/workflows/ci_build_scm_ubuntu_22.04.yml

```diff
@@ -57,33 +56,17 @@ jobs:
     #######################################################################################
     # Install FORTRAN dependencies
     #######################################################################################
-    - name: Environment for ifort compiler
-      if: contains(matrix.fortran-compiler, 'ifort')
+    - name: Install openmpi
```

Comment:
Is this going to compile openmpi with Intel if CC and FC are not set beforehand?

Reply:
I don't know. I sorta had the same question.

Reply:
I removed all the intel stuff, which wasn't being used, and never worked.

GNU is installed on the runner, so FC and CC are set accordingly there. In the case when the CI is using compilers (or wrappers) built on the server, the correct compiler environment variables need to be set. The Nvidia build/run test does this, as does this one now that we have installed compiler wrappers on the server. 

In the case of Intel, I was trying to use an old docker image provided by Intel (I think?), which I never got to work.

Reply:
Thanks for the clarification. Something for the future wish-list then!